### ISSUE TITLE
EventPipe failures in Native AOT

### ISSUE THREAD

**Issue by agocke** [dotnet/runtime#87505]
See https://github.com/dotnet/runtime/pull/83751/checks?check_run_id=14231197825 for example. Failures look like:

> ld.lld(0,0): error : undefined symbol: EventPipeWriteEventGCAllocationTick_V3

@elinor-fung Is it possible this was caused by something cross-merging with https://github.com/dotnet/runtime/pull/87465?

**Comment by elinor-fung** [comment]
> @elinor-fung Is it possible this was caused by something cross-merging with https://github.com/dotnet/runtime/pull/87465?

Yep. https://github.com/dotnet/runtime/pull/87509 should fix it.

**Comment by adamsitnik** [comment]
Fixed by #87505, thank you for a quick fix @elinor-fung !
